### PR TITLE
Fix typo in readme

### PR DESCRIPTION
**Pull request by JiyaGupta-cs**
**Summary**
I have fixed a typo in the Readme file.
compatibility  => compatible

### CODE REVIEW

**Comment by JiyaGupta-cs** [comment]
@microsoft-github-policy-service agree

@microsoft-github-policy-service agree company="Microsoft"

**Comment by JiyaGupta-cs** [comment]
@bghgary Please review and merge this fix. I am aiming for Hacktoberfest/ Hacksquad and today is the last day. Only a few hours left on 31st October.

**Comment by bghgary** [comment]
Where exactly is the typo? I don't see any changes to the text.

**Comment by JiyaGupta-cs** [comment]
Please note that this project is not at the point where updates are **fully
backwards compatibility** yet
Please note that this project is not at the point where updates are **fully backward compatible** yet

**Comment by bghgary** [comment]
This is not a typo. Both "backward compatible" and "backwards compatible" are accepted terminology.
https://en.wikipedia.org/wiki/Backward_compatibility

File: README.md [hunk 1]
```diff
@@ -40,8 +40,7 @@ Check out the [home page](https://aka.ms/Bnative) for a project overview.
 
 Babylon Native is currently available as a public preview in source form only. While many
 features are supported and functional, there are still features that are under active
-development. Please note that this project is not at the point where updates are fully
-backwards compatibility yet, and thus the contract for consuming Babylon Native can still
+development. Please note that this project is not at the point where updates are fully backward compatible yet, and thus the contract for consuming Babylon Native can still
```
Comment (bghgary) [inline README.md:43]:
```suggestion
development. Please note that this project is not at the point where updates are fully
backward compatible yet, and thus the contract for consuming Babylon Native can still
```

File: README.md [hunk 2]
```diff
@@ -115,8 +114,7 @@ the scope of what this project will support.
 
 ## Contributing
 
-Please read [CONTRIBUTING.md](./CONTRIBUTING.md) for details on our code of conduct, and 
-the process for submitting pull requests.
+Please read [CONTRIBUTING.md](./CONTRIBUTING.md) for details on our code of conduct, and the process for submitting pull requests.
```
Comment (bghgary) [inline README.md:117]:
```suggestion
Please read [CONTRIBUTING.md](./CONTRIBUTING.md) for details on our code of conduct, and
the process for submitting pull requests.
```

**Comment by JiyaGupta-cs** [comment]
compatibility  => compatible

**Comment by JiyaGupta-cs** [comment]
compatibility => compatible
@bghgary Please review and merge this fix. I am aiming for Hacktoberfest/ Hacksquad and today is the last day. Only a few hours left on 31st October.

File: README.md [hunk 3]
```diff
@@ -115,7 +115,7 @@ the scope of what this project will support.
 
 ## Contributing
 
-Please read [CONTRIBUTING.md](./CONTRIBUTING.md) for details on our code of conduct, and 
+Please read [CONTRIBUTING.md](./CONTRIBUTING.md) for details on our code of conduct, and
```
Comment (bghgary) [inline README.md:118]:
```suggestion
Please read [CONTRIBUTING.md](./CONTRIBUTING.md) for details on our code of conduct, and 
```

**Review by bghgary — APPROVED** [review]

**Pull request merged**